lib/status_code.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+require 'i18n'
```

Comment:
So, I believe this will be useless statement when we will have added `i18n` as a runtime dependency in gemspec.